src/components/MemberContribution/ContributionAccordianItem.tsx

```diff
@@ -44,7 +46,17 @@ export default function ContributionAccordianItem({
       className={styles.memberContribution_wrapper}
       data-testId="contributionContainer"
     >
-      <h3 className={styles.memberContribution_taskHeading}>{taskTitle}</h3>
+      <h3
+        className={styles.memberContribution_taskHeading}
+      >
+        {task.id ? (
+          <Link href={`${URL}${task.id}`} isExternal>
+            {taskTitle}
+          </Link>
+        ) : (
+          <Link href={`${task.url}`} isExternal>{taskTitle}</Link>
+        )}
```

Comment:
- should we render the link directly instead of nesting it inside an h3?

Reply:
Yes removed h3.